ui/app/AppLayouts/Chat/controls/community/CommunityNameInput.qml

```diff
@@ -13,11 +13,18 @@ StatusInput {
     label: qsTr("Name your community")
     charLimit: 30
     placeholderText: qsTr("A catchy name")
+
+    //validationMode: StatusInput.ValidationMode.Always
```

Comment:
leftover?

Reply:
strange... I don't have this line locally and it was pushed